bin/phpactor.php

```diff
@@ -1,38 +1,37 @@
 <?php
+use Phpactor\Console\Application;
+
+use Symfony\Component\Debug\Debug;
 
 use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Console\Output\ConsoleOutput;
```

Comment:
Moved the use declaration [because PSR](http://www.php-fig.org/psr/psr-2/#namespace-and-use-declarations), unless you are not sticking to them (I can revert)